three.c

```diff
@@ -0,0 +1,56 @@
+/*Write a program three.c that takes in 3 user-input integers and checks whether or not the 3 integers form a valid triangle (no one side is longer than the sum of the other 2 sides). It should contain the following functions:
+
+    isTriangle(), which determines whether the integers form a valid triangle
+        triangleType(), which determines whether the triangle is equilateral (all sides are equal), isoceles (only 2 sides are equal), or scalene (no sides are equal) and prints a message as appropriate 
+	sanitizedInput(), which ensures that the user-input integers are valid*/
+#include <stdio.h>
+int sanitizedInput(){
+	char line [100];
+	float x;
+	while (1) {
+//		printf("Enter a number: ");
+		fgets(line, sizeof(line), stdin);
+		if (sscanf(line, "%f", &x))
+			break; 
+		printf("Not valid, retry\n");
+		}
+		return x;
+
+}
+int isTriangle(float a,float b,float c){
+	if (a+b>c&&a+c>b&&b+c>a)
+		return 1;
+	return 0;
+
+}
+int triangleType(float a, float b, float c) {
```

Comment:
this function did return a warning - the reason for this is if the input fits neither of conditions, there's nowhere for the function to go. in other words, because there's no `else` statement, it's possible that this function has nothing to return.